nvme.c

```diff
@@ -159,6 +162,39 @@ void *nvme_alloc(size_t len, bool *huge)
 }
 #endif
 
+static int getrandom_bytes(void *buf, size_t buflen)
+{
+#if HAVE_SYS_RANDOM
+	return getrandom(buf, buflen, GRND_NONBLOCK);
+#else
+	int fd = open("/dev/urandom", O_RDONLY);
+	int result = 0, read_len;
+	if (fd < 0)
+	{
+		fd = open("/dev/random", O_RDONLY);
```

Comment:
I thought '/dev/random' is a legacy interface and you don't need to read from it on Linux. So no need
for supporting the read from blocking interface such as random (IIRC it doesn't block either on modern kernels
either.)

https://www.2uo.de/myths-about-urandom/
https://unix.stackexchange.com/questions/324209/when-to-use-dev-random-vs-dev-urandom